tests/Api/PersonalAccessTokensTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Gitlab API library.
+ *
+ * (c) Matt Humphrey <matth@windsor-telecom.co.uk>
+ * (c) Graham Campbell <hello@gjcampbell.co.uk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gitlab\Tests\Api;
+
+use Gitlab\Api\PersonalAccessTokens;
+
+class PersonalAccessTokensTest extends TestCase
+{
+    protected function getApiClass(): string
+    {
+        return PersonalAccessTokens::class;
+    }
+
+    /**
+     * @test
```

Comment:
Please use the `#[Test]` annotation instead.

Reply:
Done.